packages/dds/tree/src/shared-tree-core/messageFormat.ts

```diff
@@ -27,22 +27,21 @@ export const MessageFormatVersion = {
 	v2: 2,
 	/**
 	 * Introduced prior to 2.0 and used beyond.
-	 * Reading capability is currently maintained for backwards compatibility, but it could be removed in the future.
-	 * Writing capability needs to be maintained.
+	 * Reading capability must be maintained for backwards compatibility.
+	 * Writing capability needs to be maintained for cross-version compatibility, may be removed in FF 3.0.
```

Comment:
Thats not quite right, as we have not formally decided what the value of `lowestMinVersionForCollab` will be in FF 3.0


We could remove support for writing this format once lowestMinVersionForCollab reaches or passes the MinimumVersionForCollab in which we enable writing a newer format.

Reply:
I wasn't aware we had an explicit `lowestMinVersionForCollab`, that's nice.

Please check my reasoning here:
we'll be able to stop writing this format when we have another format that is writable in its stead (v4) and that (v4) format was readable and writable in FF versions >= `lowestMinVersionForCollab`.

Since v4 was readable and writable starting in FF v2.43.0, we'll be able to drop support for writing v3 when `lowestMinVersionForCollab` >= "2.43.0".

If that's correct, then I propose:
```suggestion
	 * Writing capability needs to be maintained for cross-version compatibility until {@link lowestMinVersionForCollab} is equal to or greater than "2.43.0".
```